package.json

```diff
@@ -32,6 +32,7 @@
     "ngrok": "3.1.1"
   },
   "devDependencies": {
+    "@types/coinstring": "^2.3.0",
```

Comment:
@SarahHouben please remove also this types 🔍

Reply:
done